app/src/staging/res/values/login_flow.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="openIdConnectBaseUrl" translatable="false">https://oidc.staging.account.gov.uk%1$s</string>
-    <string name="openIdConnectClientId" translatable="false">CLIENT_ID</string>
-    <string name="webBaseUrl" translatable="false">https://mobile-staging.account.gov.uk%1$s</string>
-</resources>
\ No newline at end of file
+    <string name="openIdConnectBaseUrl" translatable="false">https://oidc.integration.account.gov.uk%1$s</string>
+    <string name="openIdConnectClientId" translatable="false">sdJChz1oGajIz0O0tdPdh0CA2zW</string>
```

Comment:
I don't know if it matters so I'll ask anyway - are we happy with having this hardcoded within the strings / public code base?

Reply:
@elishkaflint one for you maybe?

Reply:
Thanks for flagging @Chuddington! I'm checking and will come back on this as soon as possible. I think the client id is a public identifier and not considered a secret per the spec (there are other controls eg. the client assertion - signed with a secret private key - during the token exchange) but let me double-check that.

Reply:
Confirmed that this is not considered a secret here https://gds.slack.com/archives/C04L0BZQNR3/p1699459522702629

Please feel free to resolve this convo once you've seen it and are happy @MohamedHassan101